### ISSUE TITLE
Accessibility warning

### ISSUE THREAD

**Issue by michaelcpuckett** [joerez/Woah.css#6]
You may want to provide a warning for those with seizure disorders.

https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html

Cheers!

**Comment by michaelcpuckett** [comment]
Oh, I missed it at the bottom. Thanks !

**Comment by joerez** [comment]
Thanks Michael. I'm considering removing them completely because they may be more trouble than they are worth. While they look cool, I would feel awful if it caused someone to have a seizure.
